zap/src/main/java/org/parosproxy/paros/core/scanner/Variant.java

```diff
@@ -47,6 +48,15 @@ public String setParameter(
     public String setEscapedParameter(
             HttpMessage msg, NameValuePair originalPair, String param, String value);
 
+    /**
+     * Sets the parameters into the given {@code message}.
+     *
+     * @param message the message that will be changed
+     * @param inputVectors list of name of the parameter
+     * @since TODO add version
+     */
+    default void setParameters(HttpMessage message, List<InputVector> inputVectors) {};
```

Comment:
Can be `2.11.0`, the methods added in `AbstractAppParamPlugin` should have the tag as well. Also worth adding to the new classes.

Reply:
ok